tests/conftest.py

```diff
@@ -45,6 +45,16 @@ def small_file(request):
         return join(dirname(__file__), "data", "EcoliCore.xml.gz")
 
 
+@pytest.fixture(scope="session", params=["invalid"])
```

Comment:
Why does this fixture accept parameters - it seems the invalid model is the only use case here?

Reply:
I frankly just replicated the implementation I already found, so as far as I understand this also applies to `small_file` too. Perhaps, @Midnighter could shed some light as to why this is the way it is?